tests/functional_tests/models/qwen_omni/test_qwen3_omni_conversion.py

```
@@ -0,0 +1,217 @@
+        result = subprocess.run(
+            cmd,
+            capture_output=True,
+            text=True,
+            cwd=repo_root,
+            env=env,
+        )
+
+        if result.returncode != 0:
+            print(f"STDOUT: {result.stdout}")
+            print(f"STDERR: {result.stderr}")
+            assert False, f"Qwen3-Omni conversion failed with return code {result.returncode}"
```

Ruff (0.15.9)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)